docs/logging.md

```diff
@@ -0,0 +1,204 @@
+# Contributor Guidelines - Logging
+
+## Framework
+
+We use structured logging, and specifically we use [Zap](https://github.com/uber-go/zap)
+as our logging framework.
+
+Why not something else?
```

Comment:
There's also logrus.
I don't think we need to justify which library we use, maybe down in a FAQ. IMO you can remove the `Why not something else?` section.